rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+    person.rental << self
+    book.rental << self
```

Comment:
- [ ] I think these definitions are wrong. They should be `rentals`, not `rental` based on what you defined in your respective Book and Person classes.

Reply:
Thanks, I have corrected it